frontend/packages/integration-tests-cypress/plugins/index.js

```diff
@@ -41,6 +41,23 @@ module.exports = (on, config) => {
       return null;
     },
   });
+  on('after:screenshot', (details) => {
+    // Prepend "1_", "2_", etc. to screenshot filenames because they are sorted alphanumerically in CI's artifacts dir
+    const screenshotFolder = details.path.substr(0, details.path.lastIndexOf('/'));
+    const fileName = details.path.substr(details.path.lastIndexOf('/') + 1);
```

Comment:
This probably breaks if you run the tests on Windows. It would be better to use a node utility for parsing the path.

Reply:
updated to use `path.parse`